DQMOffline/Muon/interface/MuonTiming.h

```diff
@@ -0,0 +1,85 @@
+#ifndef MuonTiming_H
+#define MuonTiming_H
+
+/** \class MuRecoAnalyzer
+ *
+ *  DQM monitoring source for muon reco track
+ *
+ *  \author G. Mila - INFN Torino
+ */
+
+#include <memory>
+#include <fstream>
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Utilities/interface/InputTag.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/ServiceRegistry/interface/Service.h"
+
+#include "DQMServices/Core/interface/DQMStore.h"
+#include "DQMServices/Core/interface/MonitorElement.h"
+#include "DQMServices/Core/interface/DQMEDAnalyzer.h"
+
+#include "RecoMuon/TrackingTools/interface/MuonServiceProxy.h"
+
+#include "DataFormats/MuonReco/interface/Muon.h"
+#include "DataFormats/MuonReco/interface/MuonFwd.h" 
+
+class MuonTiming : public DQMEDAnalyzer {
+ public:
+
+  /// Constructor
+  MuonTiming(const edm::ParameterSet&);
+  
+  /// Destructor
+  virtual ~MuonTiming();
+
+  /// Inizialize parameters for histo binning
+  void analyze(const edm::Event&, const edm::EventSetup&) override;
+  void bookHistograms(DQMStore::IBooker &, edm::Run const &, edm::EventSetup const &) override;
+
+ private:
+  // ----------member data ---------------------------
+  MuonServiceProxy *theService;
+  edm::ParameterSet parameters;
+  
+  edm::EDGetTokenT<edm::View<reco::Muon> >   theMuonCollectionLabel_;
+  // Switch for verbosity
+  std::string metname;
+    
+
+  //histo binning parameters
+  int tnbins;
```

Comment:
please follow a naming convention for variables that make up a class. Eg, in CMS its often an '_' at the end of the variable.